test/runtests.jl

```diff
@@ -1,15 +1,15 @@
-using SafeTestsets, Aqua, RasterDataSources, Pkg
+using SafeTestsets, Aqua, RasterDataSources, Pkg, Dates
```

Comment:
```suggestion
using SafeTestsets, Aqua, RasterDataSources, Pkg
```

SafeTestsets means this wont do very much